Examples/Extensions/TodayWidget/TodayViewController.h

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
nit: linter / formatter requirements?

Reply:
Good point. I've gotten used to swift lint/formatter, I'm behind on what can be leveraged in Objective-c. Will update.